how-we-work/oncall-responsibilities.md

```diff
@@ -8,7 +8,7 @@ All engineers, both frontend and backend, will be on the on-call rotation to red
 
 ## Practice and Process
 
-An on-call rotation begins on Monday at 9am PT. Each week, there is a primary and secondary on-call engineer. You will get an email and Slack notification from PagerDuty (PagerDuty manages our on-call schedule) designating you as the on-call engineer. The manager will always be on tertiary on-call duty.
+An on-call rotation begins on Monday at 9am PT/12pm ET. Each week, there is a primary and secondary on-call engineer. You will get an email and Slack notification from PagerDuty (PagerDuty manages our on-call schedule) designating you as the on-call engineer. The manager will always be on tertiary on-call duty.
```

Comment:
😄

Reply:
west coasters should be thankful I didn't place ET _first_!